lib/pulse/lib/Pulse.Lib.BoundedIntegers.fst

```diff
@@ -197,17 +197,50 @@ instance bounded_int_pos : bounded_int pos = {
 }
 
 // Using a fits predicate as the bounds check allows this class to also accomodate SizeT
-open FStar.SizeT
+
+// FIXME: Shouldn't typeclass instances be inlined at extraction? I
+// need to lift those lambdas as their standalone definitions because
+// otherwise Karamel cannot produce C code for `bounded_int_size_t`
```

Comment:
Alternatively, I could mark the class definition and the instances `inline_for_extraction`. I expect this to work for pure instances (which should be enough here), but I am not sure it will work for type classes having Pulse functions as fields. @mtzguido What do you think? Thanks in advance!

Reply:
Hi Tahina. Yes I think marking both of them as `inline_for_extraction` should do the trick. If there's any polymorphic function involved (where the class method is called over an abstract dictionary and not a concrete) you may get an exploded `match` implementing the projection, but this should be fine for non-polymorphic code.

Reply:
It works well. Thanks Guido!